src/audio/dai-zephyr.c

```diff
@@ -1599,7 +1599,7 @@ int dai_zephyr_copy(struct dai_data *dd, struct comp_dev *dev, pcm_converter_fun
 
 	/* return if nothing to copy */
 	if (!samples) {
-		comp_warn(dev, "dai_zephyr_copy(): nothing to copy");
+		comp_warn(dev, "dai_zephyr_copy(): nothing to copy, dd->total_data_processed %llu", dd->total_data_processed);
```

Comment:
Welcome addition. Might be good to add also data on the sink/src_samples status, providing hints whether we are not copying because of pipeline not providing/consuming data, or because of the DAI side. But can be done in later PR.

Reply:
on the contrary， I would suggest add less characters as possible, if there is an error here, we already get the information, we can add more information personally in local build, add too much debug related code will eventually make 1.9MB exhausted, then have to split out modules as loadable modules, then maybe more bugs will come.